gdctx/store.go

```diff
@@ -15,14 +19,25 @@ func RegisterStorePrefix(prefix string) {
 }
 
 // InitStore is to initialize the store
-func initStore() {
-	Store = store.New(Restart)
+func InitStore(initPrefix bool) {
+	Store = store.New()
```

Comment:
Begin variable names with lowercase, unless you are exporting the variable. Infact, capitalize names of variables, types, functions etc. only if you want to export them. If you are only using these within this package, keep them all in lowercase and unexported.